json_schemas/api_bridge_update_response.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": [
+    "update_available",
+    "branch_hash"
+  ],
+  "properties": {
+    "update_available": {
+      "type": "boolean"
+    },
+    "current_commit_hash": {
+      "type": "string"
+    },
+    "new_commit_hash": {
+      "type": "string"
+    },
+    "current_branch_name": {
+      "type": "string"
+    },
+    "new_branch_name": {
+      "type": "string"
+    },
+    "current_branch_release_date": {
+      "type": "string"
+    },
+    "new_branch_release_date": {
+      "type": "string"
+    },
+    "num_commits_between_branches": {
+      "type": "integer"
```

Comment:
Please add minimum of 0 (value definitions should always be as accurate as possible)

Reply:
min value set to 0